src/stores/context.store.ts

```diff
@@ -156,6 +158,7 @@ class AppContext {
         this.ensureUsers();
 
         this.context.value.selectedUserIndex = this.context.value.users!.map(e => JSON.stringify(e)).indexOf(JSON.stringify(user));
+        basketService.clear();
```

Comment:
Her clearer du stadig kurven når man skifter user. Det virker ikke særlig konsistent.

Reply:
@mzanoni De andre er ikke fjernet, de er flyttet her ind i stedet. 
Jeg havde brug for at gøre det samme i Context Switcheren som Setting siden gjorde før, så jeg flyttede det her ind i stedet således at jeg sikrer mig at den altid gør det.